<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="quit_push_button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>510</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>QUIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sr_push_button">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>510</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start/Reset</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="card_amount_spinbox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>510</y>
      <width>43</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>4</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="e_card_a_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter card amount (min 4/max 20)</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>387</width>
      <height>117</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="scoreboard_gridlayout">
     <item row="1" column="2">
      <widget class="QLabel" name="p1_round_score_label">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="p2_card_label">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="p1_match_score_label">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="ties_e_label">
       <property name="text">
        <string>Ties</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="explain_ms_label_2">
       <property name="text">
        <string>Match score</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="explain_c_label">
       <property name="text">
        <string>Cards          </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="p1_card_label">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="p2_round_score_label">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="p2_match_score_label">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="pname_label">
       <property name="text">
        <string>Enter player names</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="explain_rs_label_2">
       <property name="text">
        <string>Round score</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="p1_name_lineedit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="p2_name_lineedit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="ties_label">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="announcer_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>230</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>70</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="in_turn_label">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>In turn</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="whose_turn_label">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>370</y>
      <width>205</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="timer_gridlayout">
     <item row="1" column="2">
      <widget class="QLCDNumber" name="sec_lcd"/>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="min_lcd"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="min_label">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="sec_label">
       <property name="text">
        <string>Sec</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="timer_label">
       <property name="text">
        <string>Timer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="file_name_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>520</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Score file name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_name_lineedit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>510</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quit_push_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>841</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
